internal/tiltd/tiltd_server/tiltd.go

```diff
@@ -35,7 +36,7 @@ func NewDaemon() (*Daemon, error) {
 	return &Daemon{b: b}, nil
 }
 
-func (d *Daemon) CreateService(ctx context.Context, k8sYaml string, dockerfile string, mounts []tiltd.Mount, steps []tiltd.Cmd, dockerfileTag string) error {
+func (d *Daemon) CreateService(ctx context.Context, k8sYaml string, dockerfile string, mounts []tiltd.Mount, steps []tiltd.Cmd, dockerfileTag string, stdoutWriter io.Writer, stderrWriter io.Writer) error {
 	// TODO(maia): a real entrypoint here
 	digest, err := d.b.BuildDocker(ctx, dockerfile, mounts, steps, tiltd.Cmd{})
```

Comment:
we should prooobably be streaming the user output from the build step too -- don't need to implement now, but leave a todo?

Reply:
how about a sticky?